### PR TITLE
Em/check sort top uniqueness

### PR DESCRIPTION
**Pull request by daejunpark**

### CODE REVIEW

**Comment by daejunpark** [comment]
@cos please review

File: kernel/src/main/java/org/kframework/kore/compile/checks/CheckSortTopUniqueness.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import org.kframework.definition.SyntaxSort;
+import org.kframework.kore.Sort;
+import org.kframework.utils.errorsystem.KEMException;
+
+import static org.kframework.kore.KORE.Sort;
+
+import java.util.Set;
+
+/**
+ * Checks if multiple top sorts KList and Bag are found for
```
Comment (cos) [inline kernel/src/main/java/org/kframework/kore/compile/checks/CheckSortTopUniqueness.java]: doesn't read well. could you rephrase a bit?

File: kernel/src/main/java/org/kframework/kore/compile/checks/CheckSortTopUniqueness.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ */
+public class CheckSortTopUniqueness {
+    private final Set<KEMException> errors;
+
+    private final Module module;
+
+    public CheckSortTopUniqueness(Set<KEMException> errors, Module module) {
```
Comment (cos) [inline kernel/src/main/java/org/kframework/kore/compile/checks/CheckSortTopUniqueness.java:32]: add documentation that `errors` is a mutable Set which gets populated by this pass.

**Comment by daejunpark** [comment]
@cos please review again
